templates/tutorialv2/stats/index.html

```diff
@@ -182,6 +182,24 @@ <h3>{% trans "Échelle de temps" %}</h3>
                     </a>
                     {% endwith %}
                 </li>
+                <li>
+                    {% with start_date=db_content.public_version.publication_date.date end_date=0|datedelta_from_day %}
+                    <a class="{% if request.GET.start_date == start_date|date:'Y-m-d' %}selected{% endif %}"
+                       href="{% append_query_params start_date=start_date,end_date=end_date %}{% if display == 'comparison' %}{% for url in urls %}&urls={{url.url}}{% endfor %}{% endif %}">
+                        {% trans "Depuis la dernière publication" %}
+                    </a>
+                    {% endwith %}
+                </li>
+                {% if db_content.first_publication_date.date != public_version.publication_date.date %}
```

Comment:
J'ai mis à jour la bêta mais ça ne fonctionne pas, je pense avoir trouvé l'erreur :) (j'ai comparé avec la ligne 186)

```suggestion
                {% if db_content.first_publication_date.date != db_content.public_version.publication_date.date %}
```